src/test/java/org/folio/rest/impl/CustomFieldsImplTest.java

```diff
@@ -178,19 +245,93 @@ public void getCustomFieldsById() throws IOException, URISyntaxException {
   }
 
   @Test
-  public void deleteCustomFieldsById() {
-    deleteWithStatus(CUSTOM_FIELDS_ID_PATH, SC_NOT_IMPLEMENTED);
+  public void shouldUpdateNoteNameTypeOnPut() throws IOException, URISyntaxException {
+    final CustomField customField =
+      postWithStatus(CUSTOM_FIELDS_PATH, readFile("fields/post/postCustomField.json"), SC_CREATED, USER8)
+        .as(CustomField.class);
+    putWithNoContent(CUSTOM_FIELDS_PATH + "/" + customField.getId(), readFile("fields/put/putCustomField.json"), USER9);
+
+    CustomField field = CustomFieldsDBTestUtil.getAllCustomFields(vertx).get(0);
+    assertEquals("Department 2", field.getName());
+    assertEquals("department-_1", field.getRefId());
+    assertEquals("Provide a second department", field.getHelpText());
+    assertEquals(false, field.getRequired());
+    assertEquals(false, field.getVisible());
+    assertEquals(CustomField.Type.RADIO_BUTTON, field.getType());
+
+    final Metadata noteTypeMetadata = field.getMetadata();
+
+    assertEquals(USER8.getValue(), noteTypeMetadata.getCreatedByUserId());
+    assertEquals("m8", noteTypeMetadata.getCreatedByUsername());
+
+    assertEquals(USER9.getValue(), noteTypeMetadata.getUpdatedByUserId());
+    assertEquals("mockuser9", noteTypeMetadata.getUpdatedByUsername());
   }
 
   @Test
-  public void putCustomFieldsById() {
-    putWithStatus(CUSTOM_FIELDS_ID_PATH, "{\"name\": \"test\", \"type\": \"SINGLE_CHECKBOX\"}", SC_NOT_IMPLEMENTED);
+  public void shouldDoNotChangeRefIdWhenNameIsSameOnPut() throws IOException, URISyntaxException {
```

Comment:
Probably a typo,
it's probably meant to be shouldNotChangeRefIdWhenNameIsSameOnPut

Reply:
yes, typo, thanks @andrii-paias